OpenRA.Mods.Common/Traits/Conditions/ConditionManager.cs

```diff
@@ -76,21 +75,24 @@ void INotifyCreated.Created(Actor self)
 			state = new Dictionary<string, ConditionState>();
 			readOnlyConditionCache = new ReadOnlyDictionary<string, int>(conditionCache);
 
-			var allConsumers = new HashSet<IConditionConsumer>();
+			var allUpdaters = new HashSet<VariableObserverNotifier>();
```

Comment:
Naming nit: `allObservers` for consistency please, since we don't use "Updaters" anywhere else.